src/lib/superset-session.ts

```diff
@@ -0,0 +1,73 @@
+import axios, { AxiosInstance } from 'axios';
+import { Config } from '../config';
+
+export class SupersetSession {
```

Comment:
Are you moving https://github.com/medic/superset-user-mangement into this repo? Or are we taking it on as a dependency?